test/dataTypes.spec.ts

```diff
@@ -416,3 +436,42 @@ describe('recursive typings', function () {
     assert.ok(assertType<typeof a, Buffer>(true))
   })
 })
+
+describe('getLength', function () {
+  it('should return the right RLP length', function () {
+    assert.ok((<Buffer>RLP.getLength(0)).equals(Buffer.from([])))
+    assert.strictEqual(RLP.getLength(1), 1)
+    assert.ok((<Buffer>RLP.getLength(BigInt(0))).equals(Buffer.from([])))
+    assert.strictEqual(RLP.getLength(BigInt(1)), 1)
+  })
+})
+
+describe('stream', function () {
+  it('should handle stream decoding correctly', function () {
+    const encodedNumber = RLP.encode(1)
+    const str = 'This is a string'
+    const longString =
+      'This is a long string, so we can trigger the prefix when the buffer length is larger than 55.'
+    const encodedString = RLP.encode(str)
+    const encodedLongString = RLP.encode(longString)
+    const encodedList = RLP.encode([1, 2, 3])
+    const bufferStream = Buffer.concat([
+      encodedNumber,
+      encodedString,
+      encodedLongString,
+      encodedList,
+    ])
+    let decoded = <RLP.Decoded>(<unknown>RLP.decode(bufferStream, true))
```

Comment:
Type conversion is necessary for some reason, we should update the function signature.